web/src/components/TaskList.vue

```diff
@@ -72,25 +74,54 @@ export default {
           let data = response.data['tasks']
           for (const task in data) {
             let task_dict = data[task]
-            taskList.push({
-              task_name: task_dict.name,
-              task_id: task_dict.id,
-              task_status: task_dict.status,
-              task_success: task_dict.successful,
-              evidence_name: task_dict.evidence_name,
-              evidence_id: task_dict.evidence_id,
-              evidence_size: task_dict.evidence_size,
-            })
+            let taskStatusTemp = task_dict.status
+            // As queued status requests show as null
+            if (taskStatusTemp === null) {
```

Comment:
"pending" should be the status for queued tasks in the latest version. null could also happen, but you may want to check for "pending" here as well.

Reply:
ack updated